src/test/java/GoServletTest.java

```diff
@@ -0,0 +1,254 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.common.collect.ImmutableList;
+import com.google.maps.model.AddressType;
+import com.google.maps.model.DirectionsLeg;
+import com.google.maps.model.DirectionsResult;
+import com.google.maps.model.DirectionsRoute;
+import com.google.maps.model.Duration;
+import com.google.maps.model.GeocodingResult;
+import com.google.maps.model.Geometry;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceType;
+import com.google.maps.model.RankBy;
+import com.google.sps.model.DirectionsClient;
+import com.google.sps.model.DirectionsClientFactory;
+import com.google.sps.model.GeocodingClient;
+import com.google.sps.model.GeocodingClientFactory;
+import com.google.sps.model.PlacesClient;
+import com.google.sps.model.PlacesClientFactory;
+import com.google.sps.model.TasksClient;
+import com.google.sps.model.TasksClientFactory;
+import com.google.sps.servlets.GoServlet;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+@RunWith(JUnit4.class)
+public class GoServletTest extends AuthenticatedServletTestBase {
+
+  private DirectionsClientFactory directionsClientFactory;
+  private DirectionsClient directionsClient;
+  private PlacesClientFactory placesClientFactory;
+  private PlacesClient placesClient;
+  private TasksClientFactory tasksClientFactory;
+  private TasksClient tasksClient;
+  private GeocodingClientFactory geocodingClientFactory;
+  private GeocodingClient geocodingClient;
+  private GoServlet servlet;
+
+  private static final String API_KEY = "fake api key";
```

Comment:
nit: Keep `private` and `private static` variables together (i.e. move the `private` declarations up)

Reply:
Good idea, thanks!

Reply:
`RESOLVED`